ufl/algorithms/renumbering.py

```diff
@@ -1,87 +1,71 @@
 """Algorithms for renumbering of counted objects, currently variables and indices."""
-# Copyright (C) 2008-2016 Martin Sandve Alnæs and Anders Logg
+# Copyright (C) 2008-2024 Martin Sandve Alnæs, Anders Logg, Jø¶gen S. Dokken and Lawerence Mitchell
```

Comment:
```suggestion
# Copyright (C) 2008-2024 Martin Sandve Alnæs, Anders Logg, Jørgen S. Dokken and Lawerence Mitchell
```

Reply:
That's not how my name is spelt, but I don't need it to be here either

Reply:
Fixed now, sorry about that. I think you should be there as you wrote 99 % of that code (up to a typo).